hamlet/celery.py

```diff
@@ -330,12 +387,16 @@ def ml_subject_assistant_export_to_microsoft(
                     
                     data.append(item)
                     
-    except:
-        export.status = MLSubjectAssistantExport.FAILED
-        export.save()
-        raise
-    
+        return data
+        
+    except Exception as err:
```

Comment:
I think here you can probably just remove the try/except since it's not doing anything with the exception.

Reply:
👌